app/css/typography.css

```diff
@@ -0,0 +1,15 @@
+body {
+  background: var(--darkest-grey);
+  color: var(--light-grey);
```

Comment:
🤔 `background` and `color` into **typography.css** file?

Reply:
i pushed it around a few files and felt like they are def typography related 🤷‍♂  maybe i'll move it again? reminds me, there's other vars i typically bring around project to project.. thx!